projects/crypto-currency/destemidos/App/src/pages/SignIn/index.js

```diff
@@ -26,8 +30,46 @@ function Copyright() {
 
 export default function SignIn() {
 
+  const [inputs, setInputs] = useState({
+    email: '',
+    password: ''
+  })
+  const [open, setOpen] = useState(false)
+  const [resAPI, setResAPI] = useState('')
+  const [severity, setSeverity] = useState('')
+
+  const handleChange = (e) => {
+    inputs[e.target.name] = e.target.value
+    setInputs(inputs)
+  }
+
+  const Alert = props => {
+    return <MuiAlert elevation={6} variant="filled" {...props} />;
```

Comment:
Poderia remove o `;` ?
```suggestion
    return <MuiAlert elevation={6} variant="filled" {...props} />
```

Reply:
removido os dois pontos